group/urls.py

```diff
@@ -2,6 +2,6 @@
 from . import views
 
 urlpatterns = [
-    path("", views.group, name="group"),
+    path("profile/<int:group_id>", views.group, name="group"),
```

Comment:
If we are correcting the url with profile, let's edit "views.group" -> views.profile/views.group_profile or something. so it matches. as well as name.